R/mode.R

```diff
@@ -0,0 +1,24 @@
+check_valid_mode <- function(mode) {
+  valid_modes <- c("append", "create")
+  if (!(mode %in% valid_modes)) {
+    stop(sprintf('Invalid mode - mode must be one of %s got "%s".',
+                 paste(valid_modes, collapse = ", "), mode))
+  }
+  invisible(TRUE)
+}
+
+allow_create_table <- function(mode) {
+  allow <- FALSE
```

Comment:
Anything wrong with the body here being `mode == "create"`

Reply:
Nope - that's much nicer.